<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AstroCalculator</class>
 <widget class="QWidget" name="AstroCalculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Astro Calculator</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>85</y>
     <width>600</width>
     <height>250</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="PlanetsTab">
    <attribute name="title">
     <string>Planets</string>
    </attribute>
    <widget class="QLabel" name="PlanetAzimuth">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>120</y>
       <width>350</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>A =</string>
     </property>
    </widget>
    <widget class="QLabel" name="PlanetTransitTime">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>60</y>
       <width>350</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Transit</string>
     </property>
    </widget>
    <widget class="QLabel" name="PlanetSetTime">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>80</y>
       <width>350</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
    <widget class="QLabel" name="PlanetRightAscension">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>140</y>
       <width>350</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>alpha = </string>
     </property>
    </widget>
    <widget class="QLabel" name="PlanetAltitude">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>100</y>
       <width>350</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>h = </string>
     </property>
    </widget>
    <widget class="QLabel" name="PlanetRiseTime">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>350</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Rise</string>
     </property>
    </widget>
    <widget class="QLabel" name="PlanetDeclination">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>160</y>
       <width>350</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>delta =</string>
     </property>
    </widget>
    <widget class="QLabel" name="PlanetLabel">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>350</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Planet</string>
     </property>
    </widget>
    <widget class="QLabel" name="PlanetElongation">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>180</y>
       <width>350</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Elong =</string>
     </property>
    </widget>
    <widget class="QLabel" name="MoonRiseTime">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>80</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Rise</string>
     </property>
    </widget>
    <widget class="QLabel" name="MoonAltitude">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>100</y>
       <width>80</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>h = </string>
     </property>
    </widget>
    <widget class="QLabel" name="MoonDeclination">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>160</y>
       <width>80</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>delta =</string>
     </property>
    </widget>
    <widget class="QLabel" name="MoonTransitTime">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>80</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Transit</string>
     </property>
    </widget>
    <widget class="QLabel" name="MoonRightAscension">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>140</y>
       <width>80</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>alpha = </string>
     </property>
    </widget>
    <widget class="QLabel" name="MoonAzimuth">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>120</y>
       <width>80</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>A =</string>
     </property>
    </widget>
    <widget class="QLabel" name="MoonSetTime">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>80</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
    <widget class="QLabel" name="MoonLabel">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>80</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Moon</string>
     </property>
    </widget>
    <widget class="QLabel" name="MoonElongation">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>180</y>
       <width>80</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Elong =</string>
     </property>
    </widget>
    <widget class="QLabel" name="SunRiseTime">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>120</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Rise</string>
     </property>
    </widget>
    <widget class="QLabel" name="SunRightAscension">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>120</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>alpha = </string>
     </property>
    </widget>
    <widget class="QLabel" name="SunAltitude">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>120</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>h = </string>
     </property>
    </widget>
    <widget class="QLabel" name="SunAzimuth">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>120</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>A =</string>
     </property>
    </widget>
    <widget class="QLabel" name="SunTransitTime">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>120</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Transit</string>
     </property>
    </widget>
    <widget class="QLabel" name="SunDeclination">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>120</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>delta =</string>
     </property>
    </widget>
    <widget class="QLabel" name="SunSetTime">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>120</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
    <widget class="QLabel" name="SunLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>120</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Sun</string>
     </property>
    </widget>
    <widget class="QLabel" name="SunElongation">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>120</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Elong =</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="LocationTab">
    <attribute name="title">
     <string>Location</string>
    </attribute>
    <widget class="QDoubleSpinBox" name="longitudeInput">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>75</width>
       <height>22</height>
      </rect>
     </property>
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-180.000000000000000</double>
     </property>
     <property name="maximum">
      <double>180.000000000000000</double>
     </property>
     <property name="value">
      <double>-4.950000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="Longitude">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>70</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Longitude</string>
     </property>
    </widget>
    <widget class="QComboBox" name="citySelection">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>150</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>Turnhout</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ukkel</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Utrecht</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="Latitude">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>70</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Latitude</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="latitudeInput">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>75</width>
       <height>22</height>
      </rect>
     </property>
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-90.000000000000000</double>
     </property>
     <property name="maximum">
      <double>90.000000000000000</double>
     </property>
     <property name="value">
      <double>51.320000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="AboutTab">
    <attribute name="title">
     <string>About</string>
    </attribute>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>501</width>
       <height>151</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="NowButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Now</string>
   </property>
  </widget>
  <widget class="QLabel" name="WeekdayLabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>75</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Monday</string>
   </property>
  </widget>
  <widget class="astroDateTime" name="dateTimeEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="wrapping">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="JDLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>140</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>JD:</string>
   </property>
  </widget>
  <widget class="QLabel" name="dateLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Date/Time</string>
   </property>
  </widget>
  <widget class="QLabel" name="SiderealLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>140</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Theta:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="LocalTimecheckBox">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>50</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Local Time</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>astroDateTime</class>
   <extends>QDateTimeEdit</extends>
   <header>astroDateTime.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>NowButton</sender>
   <signal>clicked()</signal>
   <receiver>AstroCalculator</receiver>
   <slot>setDateNow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dateTimeEdit</sender>
   <signal>dateTimeChanged(QDateTime)</signal>
   <receiver>AstroCalculator</receiver>
   <slot>updateAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>citySelection</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>AstroCalculator</receiver>
   <slot>updateLongLat()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>latitudeInput</sender>
   <signal>valueChanged(double)</signal>
   <receiver>AstroCalculator</receiver>
   <slot>updateAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>153</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>longitudeInput</sender>
   <signal>valueChanged(double)</signal>
   <receiver>AstroCalculator</receiver>
   <slot>updateAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>resetDate()</slot>
  <slot>updateAll()</slot>
  <slot>updateLongLat()</slot>
  <slot>updateHorizontalCoord()</slot>
 </slots>
</ui>
